stories/assembly-aap-aws-backup-and-recovery.adoc

```diff
@@ -5,4 +5,12 @@ ifdef::context[:parent-context: {context}]
 
 :context: aap-aws-backup-recovery
 
+To backup and restore your {PlatformNameShort} deployment, it is vital to ensure automatic backups have been set for EFS and that these backups are accessible for restoration.
```

Comment:
Should you define EFS here instead of in the 2nd paragraph?

Reply:
it's actually defined at the start  of the document as well.